### ISSUE TITLE
Aegis encrypted json import error

### ISSUE THREAD

**Issue by digital-mystik** [roddhjav/pass-import#127]
Hello,

Trying out pass-import for the first time, and have been unable to successfully import an Aegis encrypted json.

Probably just a "me" issue, but everything is up-to-date on my end (python, pass, pass-import, pass-otp, etc).

Thanks

---

error message when using `pass import aegis aegis.json` (the verbose flag does not show any more relevant info):
 
`w  'str' object has no attribute 'get' [x] Error: aegis.json is not a valid exported aegis file.`

when using `pass import aegis.json`:
```Traceback (most recent call last):
  File "/usr/lib/python3.8/site-packages/pass_import/__main__.py", line 338, in pass_import
    importer.parse()
  File "/usr/lib/python3.8/site-packages/pass_import/managers/aegis.py", line 147, in parse
    self.content = self.decrypt(json.loads(self.content))
  File "/usr/lib/python3.8/site-packages/pass_import/managers/aegis.py", line 107, in decrypt
    raise ImportError(name='cryptography')
ImportError

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.8/runpy.py", line 194, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "/usr/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/usr/lib/python3.8/site-packages/pass_import/__main__.py", line 435, in <module>
    main()
  File "/usr/lib/python3.8/site-packages/pass_import/__main__.py", line 426, in main
    data = pass_import(conf, cls_import)
  File "/usr/lib/python3.8/site-packages/pass_import/__main__.py", line 353, in pass_import
    (conf['manager'], error.name, error.name))
KeyError: 'manager'

**Comment by roddhjav** [comment]
As your export file is encrypted you need the python `cryptography` module to decrypt it. That is the meaning of the first exception. The second exception is due to a bug fixed with b3bb641.

**Comment by digital-mystik** [comment]
> As your export file is encrypted you need the python `cryptography` module to decrypt it. That is the meaning of the first exception. The second exception is due to a bug fixed with [b3bb641](https://github.com/roddhjav/pass-import/commit/b3bb6415bf0c01fe48844b2d78363dc45169cbd3).

Shortly after I opened the issue I realized that I didn't have python-cryptography installed.. should have looked at the optional dependencies more carefully. Thanks for the great utility!
